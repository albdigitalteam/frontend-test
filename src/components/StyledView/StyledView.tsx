import { createBox } from "@shopify/restyle";
import { Theme } from "@theme/theme";

const StyledView = createBox<Theme>();

export default StyledView;
