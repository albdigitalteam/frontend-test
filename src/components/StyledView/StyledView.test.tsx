import React from 'react';
import { render } from '@testing-library/react-native';
import StyledView from './StyledView';

describe('snapshopt test styledview component', () => {
  it('render styledview', () => {
    const tree = render(
      <StyledView>
        <></>
      </StyledView>,
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
