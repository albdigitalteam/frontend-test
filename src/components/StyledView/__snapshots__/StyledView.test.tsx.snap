// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshopt test styledview component render styledview 1`] = `
<View
  style={
    Array [
      Object {
        "display": "flex",
      },
    ]
  }
/>
`;
